app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitController.scala

```diff
@@ -46,7 +47,8 @@ class CheckAllAnswersAndSubmitController @Inject() (
   paymentsService: PaymentsService,
   cc: MessagesControllerComponents,
   checkAllAnswersPage: pages.check_all_answers,
-  confirmationOfSubmissionPage: pages.confirmation_of_submission
+  confirmationOfSubmissionPage: pages.confirmation_of_submission,
+  val rebasingEligabilityUtil: RebasingEligabilityUtil
```

Comment:
this probably doesn't need to be a `val` - it's not overriding anything and you should be referencing it from outside the class